engine/src/main/scala/cromwell/engine/language/LanguageConfiguration.scala

```diff
@@ -0,0 +1,42 @@
+package cromwell.engine.language
+
+import java.util.Map.Entry
+
+import com.typesafe.config.ConfigFactory
+import cromwell.engine.language.CromwellLanguages.{CromwellLanguageName, CromwellLanguageVersion}
+
+import scala.collection.JavaConverters._
+
+case class LanguageConfigurationEntry(name: CromwellLanguageName, versions: Map[CromwellLanguageVersion, String])
+
+
+object LanguageConfiguration {
+  private val LanguagesConfig = ConfigFactory.load.getConfig("languages")
+  private val LanguageNames: Set[String] = LanguagesConfig.entrySet().asScala.map(findFirstKey).toSet
```

Comment:
ToL: might be a bit easier to use ficus to browse through the config instead of the built-in config methods